From 7f416c52a1e348ceb9b10d445e94fabf8857e6ff Mon Sep 17 00:00:00 2001
From: Rocky Hao <rocky.hao@rock-chips.com>
Date: Mon, 30 May 2016 09:53:29 +0800
Subject: [PATCH] ARM64: dts: rockchip: enable tsadc node for rk3366-tb

Set status of tsadc node to "okay" to make tsadc work.

Change-Id: I741adc9ce611f6f0f279fbb351dfaa5fc947db06
Signed-off-by: Rocky Hao <rocky.hao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3366-tb.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3366-tb.dts b/arch/arm64/boot/dts/rockchip/rk3366-tb.dts
index 708f2afd2fa0..15b9ded30d59 100644
--- a/arch/arm64/boot/dts/rockchip/rk3366-tb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3366-tb.dts
@@ -351,6 +351,10 @@
 	};
 };
 
+&tsadc {
+	status = "okay";
+};
+
 &nandc0 {
 	status = "okay"; /* enable both for emmc and nand */
 };
-- 
2.35.3

